src/api/sip.ts

```diff
@@ -160,6 +161,7 @@ export class SIP extends EventEmitter {
         videoSessionId: ev.videoClientSessionId,
         remoteVideoEnabled: false,
         remoteVideoStreamObject: null,
+        localVideoStreamObject: null,
       })
     })
 
```

Comment:
- de-structure object whenever required in this file
- create a function and Use switch case instead of if else in line number 177